qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -472,6 +472,15 @@ DEV=
 rbd feature disable ${POOL}/${IMAGE} journaling
 rbd config image rm ${POOL}/${IMAGE} rbd_discard_granularity_bytes
 
+# test stuck with disabling any feature
```

Comment:
```suggestion
# test that disabling a feature so that the op is proxied to rbd-nbd
# (arranged here by blkdiscard before "rbd feature disable") doesn't hang
```